xml/obs/systemsmanagement:Agama:Staging.xml

```diff
@@ -0,0 +1,15 @@
+<openQA
+    project_pattern="systemsmanagement:Agama:Staging"
+    dist_path="obspublish-other::openqa/systemsmanagement:Agama:Staging/"
+    version="agama-0.9"
+    archs="x86_64 aarch64">
+    <flavor name="ALP" folder="images/x86_64/agama-live:ALP" distri="alp">
```

Comment:
shouldn't be the flavor names prepended by agama: `agama-live:ALP` as in OBS to distinguish from other images flavors for other components? for example the cli could have also flavors, https://build.opensuse.org/package/show/systemsmanagement:Agama:Staging/agama-cli

Reply:
I just use flavor of https://openqa.opensuse.org/group_overview/96, so do you mean we need add prefix for every flavor? Such as:
ALP->agama-live:ALP
defaut->agama-live:default
default-Playwright->agama-live:default-Playwright

Reply:
> I just use flavor of https://openqa.opensuse.org/group_overview/96, so do you mean we need add prefix for every flavor? Such as: ALP->agama-live:ALP defaut->agama-live:default default-Playwright->agama-live:default-Playwright

I just use flavor of https://openqa.opensuse.org/group_overview/96, so do you mean we need add prefix for every flavor? Such as:
ALP->agama-live:ALP
defaut->agama-live:default
default-Playwright->agama-live:default-Playwright

Reply:
> shouldn't be the flavor names prepended by agama: `agama-live:ALP` as in OBS to distinguish from other images flavors for other components? for example the cli could have also flavors, https://build.opensuse.org/package/show/systemsmanagement:Agama:Staging/agama-cli That would help to distinguish them
I just use flavor of https://openqa.opensuse.org/group_overview/96, so do you mean we need add prefix for every flavor? Such as:
ALP->agama-live:ALP
defaut->agama-live:default
default-Playwright->agama-live:default-Playwright

Reply:
> shouldn't be the flavor names prepended by agama: `agama-live:ALP` as in OBS to distinguish from other images flavors for other components? for example the cli could have also flavors, https://build.opensuse.org/package/show/systemsmanagement:Agama:Staging/agama-cli That would help to distinguish them

I just use flavor of https://openqa.opensuse.org/group_overview/96, so do you mean we need add prefix for every flavor? Such as:
ALP->agama-live:ALP
defaut->agama-live:default
default-Playwright->agama-live:default-Playwright

Reply:
yes, but I guess without `:`